flavours/github-act-configure.sh

```diff
@@ -23,7 +23,7 @@ GODEBUG="asyncpreemptoff=1" /usr/local64/bin/github-act-runner configure \
     --url "https://github.com/${GITHUB_ORG}" \
     --token "${GITHUB_TOKEN}" \
     --name "${POTNAME}" \
-    --labels cheribsd,"cheribsd-${FREEBSD_VERSION}",freebsd,"freebsd-${VERSION}" \
+    --labels cheribsd,"cheribsd-${CHERIBSD_BUILD_ID}",freebsd,"freebsd-${VERSION}" \
```

Comment:
```suggestion
    --labels cheribsd,"cheribsd-${CHERIBSD_BUILD_ID}"\
```

There is very little point us adding github actions labels with FreeBSD.

Reply:
I've no objection.